src/assets/wise5/components/audioOscillator/audio-oscillator-student/audio-oscillator-student.component.ts

```diff
@@ -380,27 +316,31 @@ class AudioOscillatorController extends ComponentController {
     return firstRisingZeroCrossingIndex;
   }
 
-  isFirstRisingZeroCrossingPoint(foundFirstRisingZeroCrossing, currentY, nextY) {
+  isFirstRisingZeroCrossingPoint(
+    foundFirstRisingZeroCrossing: boolean,
+    currentY: number,
+    nextY: number
+  ): boolean {
     return !foundFirstRisingZeroCrossing && (currentY < 0 || currentY == 0) && nextY > 0;
   }
 
-  drawPoint(ctx, isFirstPointDrawn, x, y) {
+  drawPoint(ctx: any, isFirstPointDrawn: boolean, x: number, y: number): void {
     if (isFirstPointDrawn) {
       ctx.lineTo(x, y);
     } else {
       ctx.moveTo(x, y);
     }
   }
 
-  isFirstRisingZeroCrossingIndexCloseToZero(firstRisingZeroCrossingIndex) {
+  isFirstRisingZeroCrossingIndexCloseToZero(firstRisingZeroCrossingIndex: number): boolean {
     return firstRisingZeroCrossingIndex > 0 && firstRisingZeroCrossingIndex < 10;
   }
 
-  isDrawAgain() {
+  isDrawAgain(): boolean {
     return !this.stopAfterGoodDraw || (this.stopAfterGoodDraw && !this.goodDraw);
   }
```

Comment:
I *think* this the same way to write this logic?
```suggestion
  isDrawAgain(): boolean {
    return !this.stopAfterGoodDraw || !this.goodDraw;
  }
```